core/src/main/java/hudson/triggers/SCMTrigger.java

```diff
@@ -557,7 +559,7 @@ public Runner(Action[] actions) {
             if (actions == null) {
                 additionalActions = new Action[0];
             } else {
-                additionalActions = actions;
+                additionalActions = Arrays.copyOf(actions, actions.length);
```

Comment:
This technically causes an issue where if a modification to `actions` occurs outside of the ctor this class would no longer see that effect. But i don't think we support that case

Reply:
It sounds to me like it would be a bug we need to fix if the situation arises.

Reply:
I'm not sure it would be a bug here but a "wrong usage" on the other side, no?

Reply:
It should be a bug but I'm merely making a point of mentioning it.